### PR TITLE
Update pod tolerations doc to include full pod example

### PR DESCRIPTION
**Pull request by Benjamintf1**
fixes #16157

### CODE REVIEW

File: content/en/examples/pods/pod-with-toleration.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+metadata:
+  name: nginx
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
```
Comment (sftim) [inline content/en/examples/pods/pod-with-toleration.yaml]: This `nodeSelector` makes it harder for a reader to try out the sample YAML (maybe they don't have any matching nodes).

*Not* a big deal, but I wonder what the line is adding.

File: content/en/examples/pods/pod-with-toleration.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    disktype: ssd
```
Comment (sftim) [inline content/en/examples/pods/pod-with-toleration.yaml]: If keeping a `nodeSelector`, how about:
```suggestion
    kubernetes.io/os=linux
````

File: content/en/examples/pods/pod-with-toleration.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  tolerations:
+  - key: "key"
```
Comment (sftim) [inline content/en/examples/pods/pod-with-toleration.yaml]:
```suggestion
  - key: "example-key"
```

**Comment by Benjamintf1** [comment]
Updated per suggestions. I removed the node selector because it was from a previous example pod.

**Pull request merged**